.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
```

Comment:
Could add a comment with a link for where to view the results?

Reply:
Of course!

Additionally, we also have a [Scorecard badge](https://openssf.org/blog/2022/09/08/show-off-your-security-score-announcing-scorecards-badges/) that you can make more easily accessible in your README. I'd also happy to add this if you agree.

Reply:
@diogoteles08 Sure I think we can add that.

Reply:
Done!